src/plugins/intel_cpu/src/nodes/executors/fullyconnected_config.hpp

```diff
@@ -18,8 +17,6 @@ struct FCAttrs {
     // @todo probably we don't want with bias flag, since this information is already
     // a part of src memory descs
     bool withBias = false;
```

Comment:
Now I got the point. You propose avoid such bias flag and use an empty memory and empty memory desc instead. And this is a tricky part. How to recognize an empty descriptor? Check dynamic type? Null tensor or zero rank tensor are not really suitable since null tensor is a legal state of any tensor, zero rank may be used to describe a scalar.

Reply:
What about 0 rank tensor with undefined data type and undefine format (similar to oneDNN approach)

Reply:
May be simply add a corresponding "Empty" type to https://github.com/openvinotoolkit/openvino/blob/ca5a69eea01bee4fc0011bd034027f5a545bbace/src/plugins/intel_cpu/src/memory_desc/cpu_memory_desc.h#L33-L39

Reply:
Added to leftovers